build/images/base-windows/Dockerfile

```diff
@@ -0,0 +1,49 @@
+ARG GO_VERSION
+ARG NANOSERVER_VERSION
+
+FROM mcr.microsoft.com/windows/servercore:${NANOSERVER_VERSION} as windows-utility-base
+
+ARG CNI_BINARIES_VERSION
```

Comment:
This argument must place in side the new container? If not, better to place the args together?

Reply:
It must place here to be used inside windows-utility-base, otherwise the ENV variable cannot get the value.